grpc/run.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -euo pipefail
```

Comment:
In order to allow it being executed as part of test automation and to prevent users from having components being installed that incur costs, without the users understanding the full impact of them running the script, we'll need to add a cleanup routine to this script. See https://github.com/IBM/CodeEngine/blob/main/function2job/run for an example.